spec/content_tagger/updating_taxon_spec.rb

```diff
@@ -0,0 +1,40 @@
+feature "Updating a published taxon on Content Tagger", new: true, collections_publisher: true, content_tagger: true do
```

Comment:
Shouldn't the tag be just collections? collections-publisher is a different app

Reply:
@kevindew have added a new tag and we'll go through separately and update the collections tests to have it